__tests__/app.test.js

```diff
@@ -110,12 +111,63 @@ describe('/api.articles', () => {
           });
       });
 });
+describe('/api/articles/:article_id/comments', () => {
+    test('GET:200 sends an object containing the article related to the id to the client, ordered by lastest posting', () => {
+        return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({body}) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy('created_at', {
+            descending: true,
+            coerce: true
+          });
+        for (const key in comments) {
```

Comment:
As for the previous ticket, this is an index not a key because comments is an array 🙂